modules/json/tests/test_format_json.c

```diff
@@ -124,10 +124,13 @@ Test(format_json, test_format_json_with_type_hints)
                          "{\"i\":\"ifoo(\"}");
   assert_template_format("$(format-json b=boolean(TRUE))",
                          "{\"b\":true}");
+  assert_template_format("$(format-json l=list($comma_value))",
```

Comment:
Would you like to add more test cases than testing input  `value,with,a,comma`, or format-json tests are more like an integration test, while the lower layer (list-scanner) is tested thoroughly?